src/platform/site-wide/announcements/components/ScheduledMaintenance.jsx

```diff
@@ -0,0 +1,118 @@
+// Dependencies.
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment-timezone';
+import PromoBanner, {
+  PROMO_BANNER_TYPES,
+} from '@department-of-veterans-affairs/formation-react/PromoBanner';
```

Comment:
Oh actually, we have a local PromoBanner component in the same directory that has analytics hooked up.

Reply:
Oh crap I didn't see this comment lol oh well, planning on nuking this code anyways